device/arista/x86_64-arista_7280cr3mk_32p4/Arista-7280CR3-C40/gearbox_config.json

```diff
@@ -0,0 +1,334 @@
+{
+  "phys": [
+    {
+      "phy_id": 1,
+      "name": "phy1",
+      "address": "1",
+      "lib_name": "",
+      "firmware_path": "/usr/share/phy_fw.bin",
+      "config_file": "/usr/share/sonic/hwsku/phy1_config_1.json",
+      "sai_init_config_file": "",
+      "phy_access": "mdio",
+      "bus_id": 0,
+      "context_id": 1,
+      "hwinfo": "mdio0_0_0/0"
+    },
```

Comment:
@byu343  : Can you please share/update as to which azure branch you referring to to put add and validate these fields? Is it azure/master? which git commit/ reference point?

Reply:
The base sonic-buildimage image version we are using for testing is e43967645.  We plan to bring the support to master and 202106, so if any issue is from on them later, we will submit some fix.